examples/sdp3x_default_address/sdp3x_default_address.ino

```diff
@@ -9,17 +9,18 @@ void setup() {
   Wire.begin();
   Serial.begin(9600);
   delay(1000); // let serial console settle
-
-  int ret = sdp.init();
-  if (ret == 0) {
+  bool error = false;
+  do{
+    int ret = sdp.init();
+    if (ret == 0) {
       Serial.print("init(): success\n");
-  } else {
+    } else {
+	  error = true;
```

Comment:
This line mixes tabs and spaces in the indentation which is against the project style. Please use exactly 2 spaces for each indentation level.